app/views/admin/accounts/index.html.haml

```diff
@@ -53,19 +53,19 @@
         = check_box_tag :batch_checkbox_all, nil, false
       .batch-table__toolbar__actions
         - if @accounts.any?(&:user_pending?)
-          = f.button safe_join([fa_icon('check'), t('admin.accounts.approve')]),
+          = f.button safe_join([material_symbol('check'), t('admin.accounts.approve')]),
                      class: 'table-action-link',
                      data: { confirm: t('admin.reports.are_you_sure') },
                      name: :approve,
                      type: :submit
 
-          = f.button safe_join([fa_icon('times'), t('admin.accounts.reject')]),
+          = f.button safe_join([material_symbol('times'), t('admin.accounts.reject')]),
```

Comment:
Is `times` the correct icon name?
```suggestion
          = f.button safe_join([material_symbol('close'), t('admin.accounts.reject')]),
```

Reply:
> Is `times` the correct icon name?

I think when I started this branch we didn't have `close` available but we did have `times`. I see now we do have it, pushed an update to swap all the close->times changes back.